includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1108,139 +1103,139 @@ protected function get_wizard_in_cart_payment_gateways() {
 			'https://woocommerce.com/products/square/'
 		) . '</p>';
 
-		$stripe = array(
-			'name'        => __( 'Stripe', 'woocommerce' ),
-			'image'       => WC()->plugin_url() . '/assets/images/stripe.png',
-			'description' => $stripe_description,
-			'class'       => $can_stripe ? 'checked' : '',
-			'repo-slug'   => 'woocommerce-gateway-stripe',
-			'settings'    => array(
-				'create_account' => array(
-					'label'       => __( 'Create a new Stripe account for me', 'woocommerce' ),
-					'type'        => 'checkbox',
-					'value'       => 'yes',
-					'placeholder' => '',
-					'required'    => false,
-				),
-				'email' => array(
-					'label'       => __( 'Stripe email address:', 'woocommerce' ),
-					'type'        => 'email',
-					'value'       => $user_email,
-					'placeholder' => __( 'Stripe email address', 'woocommerce' ),
-					'description' => __( "Enter your email address and we'll handle account creation. WooCommerce Services and Jetpack will be installed and activated for you.", 'woocommerce' ),
-					'required'    => true,
+		return array(
+			'stripe' => array(
+				'name'        => __( 'Stripe', 'woocommerce' ),
+				'image'       => WC()->plugin_url() . '/assets/images/stripe.png',
+				'description' => $stripe_description,
+				'class'       => 'checked',
+				'repo-slug'   => 'woocommerce-gateway-stripe',
+				'settings'    => array(
+					'create_account' => array(
+						'label'       => __( 'Create a new Stripe account for me', 'woocommerce' ),
+						'type'        => 'checkbox',
+						'value'       => 'yes',
+						'placeholder' => '',
+						'required'    => false,
+					),
+					'email' => array(
+						'label'       => __( 'Stripe email address:', 'woocommerce' ),
+						'type'        => 'email',
+						'value'       => $user_email,
+						'placeholder' => __( 'Stripe email address', 'woocommerce' ),
+						'description' => __( "Enter your email address and we'll handle account creation. WooCommerce Services and Jetpack will be installed and activated for you.", 'woocommerce' ),
```

Comment:
Nitpick, but single quotes should be used here.

Reply:
String contains single quote so ok.